src/fastify-plugins/maps/styled-map-package.js

```diff
@@ -0,0 +1,113 @@
+import fp from 'fastify-plugin'
+// @ts-expect-error Need to publish types for module
+import { Reader } from 'styled-map-package'
+
+/** @import {FastifyPluginAsync} from 'fastify' */
+// @ts-expect-error Need to publish types for module
+/** @import {Resource} from 'styled-map-package/reader' */
+// @ts-expect-error Need to publish types for module
+/** @import {SMPStyle} from 'styled-map-package/types' */
```

Comment:
Just noting that these ignores are allowing tests to run without immediately failing